### PR TITLE
update metadata.json

### PR DESCRIPTION
**Pull request by J9921**
Minor Updates, More up-to-date

### CODE REVIEW

File: Bounce/dist/metadata.json [hunk 1]
```diff
@@ -11,11 +11,12 @@
     "description": {
         "en": "Bounce Radio is a 24/7 international hit music station that provides hit music to everyone. Made by gamers for gamers.",
         "nl": "Bounce Radio is een 24/7 internationaal hitmuziekstation dat iedereen hitmuziek biedt. Gemaakt door gamers voor gamers."
```
Comment (Bas950) [inline Bounce/dist/metadata.json]: Comma at the end of nl description because you put another one below it.

Comment (J9921) [reply Bounce/dist/metadata.json]: fixed.

File: Bounce/dist/metadata.json [hunk 2]
```diff
@@ -10,8 +10,7 @@
     "service": "Bounce",
     "description": {
         "en": "Bounce Radio is a 24/7 international hit music station that provides hit music to everyone. Made by gamers for gamers.",
-        "nl": "Bounce Radio is een 24/7 internationaal hitmuziekstation dat iedereen hitmuziek biedt. Gemaakt door gamers voor gamers."
-        "fr": "Bounce Radio est une station de musique à succès internationale 24/7 qui fournit de la musique à succès à tout le monde. Fabriqué par des joueurs pour des joueurs."
+        "nl": "Bounce Radio is een 24/7 internationaal hitmuziekstation dat iedereen hitmuziek biedt. Gemaakt door gamers voor gamers.",
```
Comment (Bas950) [inline Bounce/dist/metadata.json]: you changed it but removed the FR translation?!

File: Bounce/dist/metadata.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "service": "Bounce",
     "description": {
         "en": "Bounce Radio is a 24/7 international hit music station that provides hit music to everyone. Made by gamers for gamers.",
+        "fr": "Bounce Radio est une station de musique à succès internationale 24/7 qui fournit de la musique à succès à tout le monde. Fabriqué par des joueurs pour des joueurs.",
         "nl": "Bounce Radio is een 24/7 internationaal hitmuziekstation dat iedereen hitmuziek biedt. Gemaakt door gamers voor gamers.",
```
Comment (Bas950) [inline Bounce/dist/metadata.json]: please remove the comma behind the last translation.

Comment (J9921) [reply Bounce/dist/metadata.json]: Removed :)

**Pull request merged**